docs/tutorials/variational_algorithm.ipynb

```diff
@@ -11,7 +11,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
Generally it's nice to keep these at null if possible.

I think you can do this by running the notebook all the way through, saving the outputs and then clicking restart runtime without clearing the outputs.

Saved outputs in the notebook means the output present in the notebook will be used instead of running that cell in the docs infra, which this tutorial does.